app/tests/challenges_tests/test_views.py

```diff
@@ -318,3 +324,116 @@ def test_challenge_card_status(
         assert f"{expected_status} for {title}" in response.rendered_content
     else:
         assert expected_status in response.rendered_content
+
+
+@pytest.mark.django_db
+def test_challenge_request_type_2_budget_fields_required(client):
```

Comment:
This test is good, do you really need to create and verify the user to test the form? If not, see if you can find a way to remove the `django_db` marker, everything will run much quicker this way. Better place for this test would be in `test_forms`.

Reply:
The verification is indeed not necessary, but the user is I'm afraid because the ChallengeRequestForm needs a creator to be instantiated. So I think the test does need database access unfortunately, right?

Reply:
Would be strange if it needs the user saved in the db, it cannot take a user that is just built by `UserFactory.build()`?

Reply:
The form instantiation does work with `UserFactory.build()` but the validation still requires database access (I've tried `is_valid()` and `full_clean()` and `clean()`. The first two require db access, the latter complains that `cleaned_data` does not exist.

Reply:
OK then no problem!